test/client/index_test.rb

```diff
@@ -224,9 +224,12 @@
         :type            => "completion",
         :analyzer        => "simple",
         :search_analyzer => "simple",
-        :payloads        => false
       }
 
+      # COMPATIBILITY
+      # ES 5.x drops support for index-time payloads
+      suggest[:payloads] = false if es_version_2_x?
+
```

Comment:
I very much like the `COMPATIBILITY` tag here. Making a note for myself to write up the tags we are using to annotate the codebase along with what those tags should be used for.